services/src/contexts/migrations/migration_0010_delete_uploaded_datasets.rs

```diff
@@ -0,0 +1,24 @@
+use async_trait::async_trait;
+use tokio_postgres::Transaction;
+
+use crate::error::Result;
+
+use super::database_migration::{DatabaseVersion, Migration};
+
+/// This migration adds new delete options for uploaded user datasets
+pub struct Migration0011DeleteUploadedDatasets;
```

Comment:
maybe be a better name would be fair_data_deletion? otherweise one might think the migration would delete datasets...